.CI/Test/Common.c

```diff
@@ -13,6 +14,23 @@ char* ModelicaAllocateStringWithErrorReturn(size_t len) {
     return malloc(len + 1); /* Never free'd in the test programs */
 }
 
+char* ModelicaDuplicateString(const char* str) {
+    void *data = malloc(strlen(str) + 1); /* Never free'd in the test programs */
+    assert(data);
```

Comment:
The documentation comment in the header promises to call `ModelicaError` in case of failure.

Reply:
Maybe the best solution would be to use `ModelicaAllocateString` for allocation, and then make sure `ModelicaAllocateString` actually calls `ModelicaError`?

Reply:
Well, already wrong in ModelicaAllocateString.